docs/guides/dnd/DndKitMultiple/DndKitMultiple.tsx

```diff
@@ -0,0 +1,279 @@
+import { useMemo, useState, HTMLAttributes } from "react";
+import {
+  DndContext,
+  DragEndEvent,
+  DragOverEvent,
+  DragOverlay,
+  DragStartEvent,
+  KeyboardSensor,
+  PointerSensor,
+  useSensor,
+  useSensors,
+  Modifier,
+} from "@dnd-kit/core";
+import { arrayMove, SortableContext, useSortable } from "@dnd-kit/sortable";
+import { CSS } from "@dnd-kit/utilities";
+import { restrictToWindowEdges } from "@dnd-kit/modifiers";
+import {
+  HvListContainer,
+  HvListItem,
+  HvTypography,
+  theme,
+  useTheme,
+} from "@hitachivantara/uikit-react-core";
+import { Drag } from "@hitachivantara/uikit-react-icons";
+
+import { css } from "@emotion/css";
+
+import { sampleColumns, sampleItems } from "./sampleData";
+import { Column, Item } from "../types";
+import classes from "./styles";
+
+// Fixes a problem we have while dragging node types from the sidebar to the flow in storybook docs mode
```

Comment:
We should update the comment since it doesn't apply to this sample

Same for DndKitSingle